Source/AnimatableItem.swift

```diff
@@ -10,6 +10,42 @@ import UIKit
 
 public protocol AnimatableItem: AnyObject {
     var position: CGPoint { get }
+    var scale: CGFloat { get }
+    var scaleX: CGFloat { get }
+    var scaleY: CGFloat { get }
+    var cgOpacity: CGFloat { get }
+    var rotation: CGFloat { get }
 }
 
-extension CALayer: AnimatableItem { }
+extension CALayer: AnimatableItem {
+    public var scale: CGFloat {
+        let presentationLayer = self.presentation() ?? self
+        _ = presentationLayer.affineTransform() // let transform
```

Comment:
`// let transform` i think this was not intended
Also I would recommend using `///` for commentary instead of `//`

Reply:
Yeah that's a lot of comments to be changed, but it is fairly easy to do it with find-replace across the project using Xcode